lib/beaker/host/unix.rb

```diff
@@ -58,12 +62,16 @@ def self.foss_defaults
 
     def self.aio_defaults
       h = self.foss_defaults
-      h['puppetbindir'] = '/opt/puppetlabs/agent/bin'
-      h['puppetpath'] = '/etc/puppetlabs/agent'
-      h['puppetvardir'] = '/opt/puppetlabs/agent/cache'
-      h['distmoduledir'] = '/opt/puppetlabs/agent/modules'
-      h['sitemoduledir'] = '/etc/puppetlabs/agent/modules'
-      h['hieraconf'] = ' /etc/puppetlabs/agent/hiera.conf'
+      h['puppetbindir']   = '/opt/puppetlabs/agent/bin'
+      h['puppetbin']      = "#{h['puppetbindir']}/puppet"
+      h['puppetpath']     = '/etc/puppetlabs/agent'
+      h['puppetconfdir']  = "#{h['puppetpath']}/config"
+      h['puppetcodedir']  = "#{h['puppetpath']}/code"
```

Comment:
We need to add this property to the windows host too, it should have the same value as `puppetconfdir`